vizro-core/examples/visual-vocabulary/pages/distribution.py

```diff
@@ -79,58 +80,63 @@
 
 butterfly = butterfly_factory("distribution")
 
```

Comment:
This is a very confusing diff (not your fault). Is it just switching the order of `dumbell` and `histogram`?

Reply:
Not sure to be honest 😄 The only thing I changed was to add the dumbbell to the bottom of the page, as I always add new charts to the bottom.